nautobot/utilities/tests/test_utils.py

```diff
@@ -215,3 +217,33 @@ def test_is_truthy(self):
         self.assertFalse(is_truthy("n"))
         self.assertFalse(is_truthy(0))
         self.assertFalse(is_truthy("0"))
+
+
+class PrettyPrintQueryTest(TestCase):
+    """Tests for `pretty_print_query()."""
+
+    def test_pretty_print_query(self):
```

Comment:
Would be good to also test with Qs that have more than a single parameter, e.g `Q(site__slug="ams01", status__slug="planned", ...)`